sales_contact_finder/src/tools/api_poster.py

```diff
@@ -0,0 +1,71 @@
+from crewai.tools import BaseTool
+from pydantic import Field
+import requests
+import json
+
+class ContactPosterTool(BaseTool):
+    name: str = "ContactPosterTool"
+    description: str = """
+    Posts contact information to an API endpoints. Expected payload format:
```

Comment:
maybe make this a little more specific to Gibson?
```suggestion
    Saves contact information in a GibsonAI database using the hosted API. Expected payload format:
```